### PR TITLE
Store admin

### PR DESCRIPTION
**Pull request by adaline**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: 2.0
 jobs:
   test:
     docker:
-      - image: circleci/ruby:2.7-node
+      - image: circleci/ruby:2.7.0-node
```
Comment (raen79) [inline .circleci/config.yml:6]: why?

File: app/controllers/admin/trade_stores_controller.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        ::Store.page(params[:page]).per(params[:per_page] || Spree::Config[:orders_per_page])
+    end
+
+    def new
+      @store = ::Store.new
+    end
+
+    def create
+      @store = ::Store.new(store_params)
+      if @store.save
```
Comment (raen79) [inline app/controllers/admin/trade_stores_controller.rb:16]: I'm a nut but put a line break between `@store = ...` and `if @store.save` please

File: app/controllers/admin/trade_stores_controller.rb [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      else
+        flash.now[:error] = @store.errors.full_messages.join(', ')
+        render :edit, status: :unprocessable_entity
+      end
+    end
+
+    private
+
+    def model_class
+      ::Store
```
Comment (raen79) [inline app/controllers/admin/trade_stores_controller.rb:42]: this doesn't look like it's used anywhere

File: app/controllers/admin/trade_stores_controller.rb [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        :street,
+        :street_number,
+        :city,
+        :country,
+        :phone_number,
+        :is_hub
+      )
+    end
+
+    # rubocop:enable Metrics/MethodLength
```
Comment (raen79) [inline app/controllers/admin/trade_stores_controller.rb:60]: just remove this on an application level.

File: .circleci/config.yml (hunk 1 above)
Comment (adaline) [reply .circleci/config.yml:6]: https://github.com/raen79/apocalypse-delivery/blob/master/Gemfile#L6

```
Your Ruby version is 2.7.1, but your Gemfile specified 2.7.0

Exited with code exit status 18
CircleCI received exit code 18
```

File: app/controllers/admin/trade_stores_controller.rb (hunk 4 above)
Comment (adaline) [reply app/controllers/admin/trade_stores_controller.rb:60]: You sure you dont want that sweet sweet rubocop coding style goodness?!

File: app/controllers/admin/trade_stores_controller.rb (hunk 3 above)
Comment (adaline) [reply app/controllers/admin/trade_stores_controller.rb]: well spotted

File: app/controllers/admin/trade_stores_controller.rb (hunk 2 above)
Comment (adaline) [reply app/controllers/admin/trade_stores_controller.rb:17]: heh, its fine

File: app/controllers/admin/trade_stores_controller.rb (hunk 3 above)
Comment (adaline) [reply app/controllers/admin/trade_stores_controller.rb]: Oh no, seems it is need after all:

```Stores Admin new store creation creates a new Store

     Failure/Error: visit '/admin/trade_stores/new'

     

     NameError:

       uninitialized constant Spree::TradeStore

     # ./spec/features/spree/admin/stores_spec.rb:44:in `block (3 levels) in <top (required)>'
```

File: app/controllers/admin/trade_stores_controller.rb (hunk 4 above)
Comment (raen79) [reply app/controllers/admin/trade_stores_controller.rb:61]: fine leave it haha

**Pull request merged**